Dockerfile

```diff
@@ -29,7 +27,7 @@ COPY main.go main.go
 COPY controllers/ controllers/
 COPY internal/ internal/
 
-# build without giving the arch, so that it gets it from the machine
+# build
```

Comment:
Probably good to add `-trimpath` to `xx-go build` now that we're making changes to the file anyway.